modules/govuk_jenkins/manifests/job/deploy_efg.pp

```diff
@@ -0,0 +1,11 @@
+# == Class: govuk_jenkins::job::deploy_efg
+#
+# Create a file on disk that can be parsed by jenkins-job-builder
+#
+class govuk_jenkins::job::deploy_efg {
+  file { '/etc/jenkins_jobs/jobs/deploy_efg.yaml':
+    ensure  => present,
+    content => template('govuk_jenkins/jobs/deploy_efg.yaml.erb'),
+    notify  => Exec['jenkins_jobs_update'],
+  }
```

Comment:
Would it be better to include this file resource as part of the `govuk_jenkins::job::deploy_app` class? Otherwise this class will have to be included in the list of Jenkins jobs.

There are lots of variables used in the template like `@ci_deploy_jenkins_api_key` which is undefined here.